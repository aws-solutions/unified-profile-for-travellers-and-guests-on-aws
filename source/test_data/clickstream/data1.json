{"modelVersion":"1.0","event_type":"SearchExperience","event_timestamp":"2022-01-15T06:22:28.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[2.0,3.0,1.0,5.0,1.0,5.0,1.0,2.0,1.0,2.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["DBL-BAR","CHMPGN","BKFST","KING_SUPER_SAVER","SUITE-BAR","CHMPGN","KING_SUPER_SAVER","SUITE-BAR","SUITE-BAR","SUITE-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"DBL","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"PAR03","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["ROM01","LIM16","TOK00","LIM13","NYC03","LON07","ROM11","NYC00","LIM00","LIM07"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"SelectStartDate","event_timestamp":"2022-01-15T06:22:38.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[1.0,2.0,5.0,4.0,5.0,1.0,5.0,5.0,4.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","KING_SUPER_SAVER","CHMPGN","CHMPGN","SUITE-BAR","CHMPGN","KING_SUPER_SAVER","DBL-BAR","KING_SUPER_SAVER","KING_SUPER_SAVER"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"SUPER_SAVER","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"SGP17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["TOK18","ATL08","LIM03","LON03","TOK05","ATL18","TOK00","ROM13","TOK08","LIM13"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"ViewProduct","event_timestamp":"2022-01-15T06:22:39.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[2.0,5.0,3.0,2.0,1.0,5.0,4.0,3.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["DBL-BAR","SUITE-BAR","CHMPGN","DBL-BAR","SUITE-BAR","BKFST","CHMPGN","DBL-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"LON08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["TOK11","SGP12","LON04","LIM12","LIM13","PAR15","LIM00","NYC16"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"ViewProduct","event_timestamp":"2022-01-15T06:22:50.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[1.0,5.0,5.0,4.0,1.0,5.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","KING_SUPER_SAVER","DBL-BAR","DBL-BAR","SUITE-BAR","CHMPGN"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"LIM00","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["TOK06","ATL04","PAR01","TOK06","PAR11","ATL05"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"RetreiveBooking","event_timestamp":"2022-01-15T06:23:05.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[3.0,4.0,4.0,2.0,3.0,3.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["BKFST","CHMPGN","BKFST","DBL-BAR","BKFST","CHMPGN"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"ROM07","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["ATL09","ROM07","ATL11","PAR10","SGP18","ROM15"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"IgnoreBooking","event_timestamp":"2022-01-15T06:23:12.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[1.0,5.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["DBL-BAR","CHMPGN","KING_SUPER_SAVER"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"SGP14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["LON05","PAR09","LIM17"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"custom","event_timestamp":"2022-01-15T06:23:19.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[4.0,2.0,1.0,5.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","KING_SUPER_SAVER","DBL-BAR","SUITE-BAR","SUITE-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"SUPER_SAVER","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"PAR14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["TOK10","SPO18","NYC12","LON11","NYC12"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"StartSession","event_timestamp":"2022-01-15T06:23:31.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[4.0,1.0,3.0,2.0,4.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["SUITE-BAR","BKFST","SUITE-BAR","BKFST","KING_SUPER_SAVER"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"SGP17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["NYC06","LIM18","SPO01","LIM14","PAR02"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
{"modelVersion":"1.0","event_type":"SelectProduct","event_timestamp":"2022-01-15T06:23:43.899545599Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"361fa48f-1085-4116-bc81-413b34747f58"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[3.0,5.0,2.0,4.0,5.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["BKFST","DBL-BAR","BKFST","SUITE-BAR","DBL-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Denver, Tennessee, French Polynesia","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":2.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"DBL","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2024-02-15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2024-02-17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"TOK15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["ROM10","ATL06","PAR06","NYC06","NYC10"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}
