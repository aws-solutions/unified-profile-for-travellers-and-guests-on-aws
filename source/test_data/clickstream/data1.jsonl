{"modelVersion":"1.0","event_type":"search_experience","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36","ipAddress":"27.18.77.197"},"session":{"id":"88941775-7c26-437a-a061-84996a448f50"},"attributes":[{"type":"string","name":"customer_email","stringValue":"jamiehintz@example.com"},{"type":"string","name":"customer_phone","stringValue":"22 211.907.3813"},{"type":"string","name":"loyalty_id","stringValue":"3051056168"},{"type":"string","name":"customer_gender","stringValue":"female"},{"type":"string","name":"customer_id","stringValue":"8375504656"},{"type":"string","name":"language_code","stringValue":"sv"},{"type":"string","name":"customer_birthdate","stringValue":"2008-08-21"},{"type":"string","name":"customer_first_name","stringValue":"Kaela"},{"type":"string","name":"customer_last_name","stringValue":"Glover"},{"type":"numbers","name":"quantities","numValues":[3.0,1.0,2.0,5.0,1.0,2.0,1.0,4.0]},{"type":"strings","name":"products","stringValues":["SEAT","FARE","LUGG","SEAT","SEAT","FARE","LUGG","LUGG"]},{"type":"numbers","name":"products_prices","numValues":[431.0,153.0,796.0,586.0,105.0,197.0,616.0,133.0]},{"type":"string","name":"fare_class","stringValue":"F"},{"type":"string","name":"fare_type","stringValue":"N"},{"type":"strings","name":"flight_segments_departure_date_time","stringValues":["2025-08-28T02:52:28","2025-08-28T07:52:28","2025-08-28T18:52:28","2025-08-29T07:52:28"]},{"type":"strings","name":"flight_segments_arrival_date_time","stringValues":["2025-08-28T06:52:28","2025-08-28T16:52:28","2025-08-29T05:52:28","2025-08-29T17:52:28"]},{"type":"strings","name":"flight_segment_sku","stringValues":["DL9120","DL6022","DL6747","DL8983"]},{"type":"string","name":"flight_market","stringValue":"DOM,DOM"},{"type":"string","name":"flight_type","stringValue":"round-trip"},{"type":"string","name":"origin_date","stringValue":"DOM,DOM"},{"type":"string","name":"origin_date_time","stringValue":"DOM,DOM"},{"type":"string","name":"returning_date_time","stringValue":"DOM,DOM"},{"type":"string","name":"returning_date","stringValue":"DOM,DOM"},{"type":"string","name":"returning_flight_route","stringValue":"DOM,DOM"},{"type":"number","name":"num_pax_adults","numValue":4.0},{"type":"number","name":"num_pax_inf"},{"type":"number","name":"num_pax_children"},{"type":"string","name":"pax_type","stringValue":"adult"},{"type":"number","name":"total_passengers","numValue":3.0}],"endpoint":{},"alternateProfileIds":[{"name":"mob","value":"51a7df44-2174-4455-9da0-ad4993d222b5","description":"Quod velit impedit rerum id.","lastUpdatedOn":"1972-01-30T19:51:14.430846Z"},{"name":"truth","value":"b6f6f2af-cc79-4f25-8f15-fa84c11b6dfc","description":"Rerum praesentium eos ipsa quia.","lastUpdatedOn":"1936-08-15T10:50:04.746945Z"}],"event_timestamp":"2024-09-27T15:15:36.491166Z","arrival_timestamp":"0001-01-01T00:00:00.000000Z"}
{"modelVersion":"1.0","event_type":"cross_geofence","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36","ipAddress":"102.237.83.104"},"session":{"id":"88941775-7c26-437a-a061-84996a448f50"},"attributes":[{"type":"string","name":"geofence_latitude","stringValue":"32.082924"},{"type":"string","name":"geofence_longitude","stringValue":"120.062928"},{"type":"string","name":"geofence_name","stringValue":"Restaurant"},{"type":"string","name":"geofence_id","stringValue":"restaurant_1"}],"endpoint":{},"event_timestamp":"2024-09-27T15:15:53.491166Z","arrival_timestamp":"0001-01-01T00:00:00.000000Z"}
