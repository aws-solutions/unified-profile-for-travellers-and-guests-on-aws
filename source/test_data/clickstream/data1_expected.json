{"model_version": "1.0", "object_type": "clickstream_event", "event_type": "SearchExperience", "event_timestamp": "2022-01-15T06:22:28.899545599Z", "arrival_timestamp": 0, "event_version": "", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36", "session_id": "361fa48f-1085-4116-bc81-413b34747f58", "traveller_id": "361fa48f-1085-4116-bc81-413b34747f58", "destination": "Denver, Tennessee, French Polynesia", "checkin_date": "2024-02-15", "products": "DBL-BAR|CHMPGN|BKFST|KING_SUPER_SAVER|SUITE-BAR|CHMPGN|KING_SUPER_SAVER|SUITE-BAR|SUITE-BAR|SUITE-BAR", "num_nights": 2.0, "room_type": "DBL", "rate_plan": "BAR", "checkout_date": "2024-02-17", "hotel_code": "PAR03", "hotel_code_list": "ROM01|LIM16|TOK00|LIM13|NYC03|LON07|ROM11|NYC00|LIM00|LIM07", "aws_account_id": "", "last_updated": "2022-01-15T06:22:28.899545599Z", "last_updated_partition": "2022/01/15/06/22/28/899545599Z"}